cves/CVE-2017-7659.yml

```diff
@@ -47,7 +47,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  HTTP/2 requests could be crafted in a way that could cause mod_http2 to
+  dereference a NULL pointer and crash the server process.
```

Comment:
Some more information about how the HTTP/2 request could cause the server to dereference a null pointer could be helpful. How does the `mod_http2` module parse the request incorrectly?